src/Gang/ui/TerritorySubpage.tsx

```diff
@@ -85,9 +85,11 @@ export function TerritorySubpage(): React.ReactElement {
         </Typography>
       </Box>
       <Box sx={{ display: "grid", gridTemplateColumns: "repeat(3, 1fr)" }}>
-        {gangNames.map((name) => (
-          <OtherGangTerritory key={name} name={name} />
-        ))}
+        {gangNames
+          .filter((name) => AllGangs[name].territory > 0)
```

Comment:
I think it might be a bit unpleasant from a UI perspective to remove defeated gangs entirely.

I might suggest something like sorting defeated gangs to the back, and lowering their element opacity (like how it's done in the `Create Program` page

```suggestion
          .sort((a, b) => {
            if (AllGangs[a].territory <= 0) return 1;
            if (AllGangs[b].territory <= 0) return -1;
            return 0;
          })
```

And then reduce the opacity by like 25% for enemy gangs with 0 territory

![image](https://user-images.githubusercontent.com/60761231/167320973-03527717-c366-4ac7-8614-2d6fddf183cf.png)